app/controllers/concerns/curation_concerns/collections_controller_behavior.rb

```diff
@@ -13,6 +13,9 @@ module CollectionsControllerBehavior
 
       # This is needed as of BL 3.7
       copy_blacklight_config_from(::CatalogController)
+      configure_blacklight do |config|
+        config.search_builder_class = CurationConcerns::CollectionSearchBuilder
```

Comment:
@jcoyne: see here https://github.com/projecthydra/curation_concerns/blob/master/app/controllers/concerns/curation_concerns/selects_collections.rb#L7-L9

I'm happy enough to delete it, but it seems like it was required in that other case at least.  Should it be deleted in both?